snapcraft.yaml

```diff
@@ -1,79 +1,48 @@
 name: brave
 version: 1.18.70
-base: core18
+grade: stable
+architectures:
+  - build-on: amd64
 summary: Browse faster and safer with Brave.
 description: |
  The new Brave browser automatically blocks ads and trackers, making it
  faster and safer than your current browser. Start supporting publishers
  today with our new Brave Payments system.
-
-grade: stable
 confinement: strict
-
-architectures:
-  - build-on: amd64
-
-parts:
-  brave:
-    plugin: dump
-    source: https://github.com/brave/brave-browser/releases/download/v$SNAPCRAFT_PROJECT_VERSION/brave-browser_$SNAPCRAFT_PROJECT_VERSION_amd64.deb
-    source-type: deb
-    # Correct path to icon.
-    override-pull: |
-      snapcraftctl pull
-      rm -rf etc/cron.daily/
-      rm -rf usr/bin/brave-browser usr/bin/brave-browser-stable
-      chmod 0755 ./opt/brave.com/brave/chrome-sandbox
-      sed -i 's|Icon=brave-browser|Icon=/opt/brave.com/brave/product_logo_128\.png|g' usr/share/applications/brave-browser.desktop
-      sed -i -e '/^Actions=/d' usr/share/applications/brave-browser.desktop
-    after:
-      - desktop-gtk3
-    stage-packages:
-      - gir1.2-gnomekeyring-1.0
-      - libasound2
-      - libgconf-2-4
-      - libgl1-mesa-glx
-      - libglu1-mesa
-      - libgnome-keyring0
-      - libcap2
-      - libgcrypt20
-      - libnotify4
-      - libnspr4
-      - libnss3
-      - libpulse0
-      - libxtst6
-      - libxss1
-
+base: core18
 apps:
   brave:
     command: bin/desktop-launch $SNAP/opt/brave.com/brave/brave-browser
     desktop: usr/share/applications/brave-browser.desktop
-    # Correct the TMPDIR path for Chromium Framework/Electron to
-    # ensure libappindicator has readable resources.
     environment:
-      # Correct the TMPDIR path for Chromium Framework/Electron to
-      # ensure libappindicator has readable resources
-      TMPDIR: $XDG_RUNTIME_DIR
-      # Fallback to XWayland if running in a Wayland session.
       DISABLE_WAYLAND: 1
+      BRAVE_DESKTOP: brave-browser.desktop
+      BRAVE_CONFIG_HOME: $SNAP_USER_COMMON
     plugs:
-      - avahi-observe
```

Comment:
Might be worth testing to see if you can see local printers without avahi.

Reply:
I don't have the means to test this. I can follow up in a further QA ticket.